src/components/Datatable/Pagination/Pagination.tsx

```diff
@@ -144,9 +144,7 @@ const Pagination: React.FC<PaginationProps> = ({
           aria-label="Go to Page #"
           placeholder="#"
           type="number"
-          onChange={(event) => {
-            onChangePageNumber(event);
-          }}
+          onKeyDown={(e) => handlePageInput(e)}
```

Comment:
Very minor, I would rename `e` to `event` to be unified and to prevent abbreviations